src/dotnet/UI.Blazor.App/Components/ChatView/Items/Conversation/conversation.css

```diff
@@ -155,6 +155,8 @@
     scrollbar-width: none;
 }
 
-.virtual-list > ul.c-virtual-container > li.item:has(.conversation-message) {
+.virtual-list > .c-wrapper > ul.c-virtual-container > li.item:has(.conversation-message),
+.virtual-list > .c-wrapper > ul.c-virtual-container > li.item:has(.conversation-header),
```

Comment:
Why do we need selectors with `conversation-header` and `conversation-footer`. They are always inside `conversation-message`. Or I misunderstood something?